### ISSUE TITLE
Improperly parsed literals

### ISSUE THREAD

**Issue by ghost** [slynch8/10x#1320]
![Capture](https://user-images.githubusercontent.com/121533901/209805982-7be58b7b-338f-4dad-a945-b674f37b2e33.PNG)

```c++
0xFED7'7798'7654'3210;
0xFEDC'BA98'7654'3210;
0x1.1A62633145C07p-54;
// autocorrected into
0x1.1 A62633145C07p-54;	
```

**Comment by slynch8** [comment]
I've fixed the apostrophes, but is `0x1.1 A62633145C07p-54` even a valid float?

**Comment by slynch8** [comment]
fixed (0.35.77)
